### PR TITLE
Close toss, and some other minor enhancement

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [X] enhancement

#### Issue(s) number: 
* Close https://github.com/vesoft-inc/nebula-ent/issues/1857 : disable toss
* Close https://github.com/vesoft-inc/nebula-ent/issues/1738 https://github.com/vesoft-inc/nebula-ent/issues/1953: add query_concurrently to conf, remove kv_separation from conf because rocksdb has bug
* Close https://github.com/vesoft-inc/nebula-ent/issues/1954, https://github.com/vesoft-inc/nebula-ent/issues/1978: rename listener's pid_file name and log_dir name
* Close https://github.com/vesoft-inc/nebula-ent/issues/1797: make default value of `local_config` as true
* Some other minor enhancement, disable some useless tools.

## Checklist:
Tests:
- [X] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: tests/tck/features/match/With.feature [hunk 1]
```diff
@@ -325,7 +325,7 @@ Feature: With clause
     Given an empty graph
     And having executed:
       """
-      CREATE SPACE IF NOT EXISTS `nebula_huskie` (partition_num = 32, replica_factor = 1, charset = utf8, collate = utf8_bin, vid_type = INT64, atomic_edge = false);
+      CREATE SPACE IF NOT EXISTS `nebula_huskie` (partition_num = 32, replica_factor = 1, charset = utf8, collate = utf8_bin, vid_type = INT64);
```
Comment (liwenhui-soul) [inline tests/tck/features/match/With.feature:328]: does parser also need to remove? 
"KW_ATOMIC_EDGE ASSIGN BOOL"

**Review by SuperYoko — DISMISSED** [review]

File: tests/tck/features/match/With.feature (hunk 1 above)
Comment (critical27) [reply tests/tck/features/match/With.feature:328]: 👌

File: src/tools/storage-perf/CMakeLists.txt [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
-    LIBRARIES
-        ${ROCKSDB_LIBRARIES}
-        ${THRIFT_LIBRARIES}
-        ${PROXYGEN_LIBRARIES}
-        gtest
-        curl
-)
+#nebula_add_executable(
+#    NAME
+#        storage_perf
```
Comment (codesigner) [inline src/tools/storage-perf/CMakeLists.txt:7]: why comment out this tools (simple_kv_verify, storage_perf, storage_integrity), is it related that toss is turned off?

**Review by codesigner — COMMENTED** [review]
LGTM,

**Comment by critical27** [comment]
TCK failed, hold on a sec.

File: src/tools/storage-perf/CMakeLists.txt (hunk 2 above)
Comment (critical27) [reply src/tools/storage-perf/CMakeLists.txt:7]: Not related to toss, these tools are out of date now, which has not been maintained for a long time.

**Comment by critical27** [comment]
Ready now, verified in local.

**Review by liwenhui-soul — APPROVED** [review]

**Comment by Sophie-Xie** [comment]
How about the query_concurrently flag? 😂

**Comment by critical27** [comment]
> How about the query_concurrently flag? 😂

Added, with default value false

**Comment by wenhaocs** [comment]
Awesome!

**Review by wenhaocs — APPROVED** [review]

**Comment by wenhaocs** [comment]
> > How about the query_concurrently flag? 😂
> 
> Added, with default value false

I see argument about this in the ticket. Please confirm.

**Comment by critical27** [comment]
> > > How about the query_concurrently flag? 😂
> > 
> > 
> > Added, with default value false
> 
> I see argument about this in the ticket. Please confirm.

After discuss, will make it true by default

**Review by liwenhui-soul — APPROVED** [review]

**Review by wenhaocs — APPROVED** [review]

**Review by codesigner — APPROVED** [review]

**Pull request merged**